content/en/dashboards/guide/powerpacks-best-practices.md

```diff
@@ -22,7 +25,8 @@ Powerpacks are templated groups of widgets that scale graphing expertise as reus
 Powerpacks are either preset (created by Datadog) or custom (created by a user).
 
 - Preset Powerpacks provide out-of-the-box views for common monitoring patterns like performance metrics or feature usage. They are often linked to a specific product or integration (like `RUM Page Views`) and are maintained by Datadog.
-- Custom Powerpacks can be created by anyone with dashboard write permissions to help users share and standardize internal best practices.
+- Custom Powerpacks can be created by anyone with dashboard write permissions to help users share and standardize internal best practices. Custom powerpacks are synced, and layout changes will update all instances where the powerpack is used.
```

Comment:
```suggestion
- Anyone with dashboard write permissions can create custom Powerpacks to help users share and standardize internal best practices. Custom powerpacks are synced, and layout changes will update all instances where the powerpack is used.
```

Reply:
> Custom powerpacks are synced,

By what or who? This is passive voice, but I am not sure how to correct because I am missing context.

Reply:
> layout changes will update

this is passive too